.infrastructure/4_rds.tf

```diff
@@ -0,0 +1,41 @@
+resource "random_string" "random" {
+  length  = 6
+  special = false
+  upper   = false
+}
```

Comment:
I think this resource is useless

Reply:
Correct, I'll remove it from the code with the next commit